stylelint.config.cjs

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  extends: [
+    '@extensionengine/stylelint-config'
+  ],
+  overrides: [
+    {
+      files: ['src/**', 'example/**'],
+      rules: {
+        'selector-pseudo-class-no-unknown': null,
```

Comment:
What pseudo-class selectors are we using that require to unset this rule.

Reply:
Removed.